week_2/time-master/function.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+export function renderAlarm(list, alarms) {
+	list.innerHTML = "";
+
+	const setAlarms = alarms
+		.map(
+			(alarm) =>
+				`
```

Comment:
install a proper linter for code

Reply:
Edited but can you recommend one. I use Prettier